libs/dal/src/repositories/execution-details/execution-details.schema.ts

```diff
@@ -112,8 +111,6 @@ executionDetailsSchema.index({
   _environmentId: 1,
 });
 
-executionDetailsSchema.index({ expireAt: 1 }, getTTLOptions());
```

Comment:
Small note, let's make sure we remove this index from the Atlas console after releasing to Prod.

Reply:
![Captain](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExN2ppdWw1dG51bTVkazVpMnBvNTZxbWI2NTN4MWIwcmVjdWQyZ3lqMyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/l1AsWIgMMGvziKgMg/giphy.gif)